base_station/gui/src/components/AutonTask.vue

```diff
@@ -121,10 +121,8 @@ export default {
         distance: 0
       },
 
-      TargetList: {
-        targetList: [{bearing: 0, distance: 0, id: 0},
-        {bearing: 0, distance: 0, id: 0}]
-      },
+      TargetList: [{bearing: 0, distance: 0, id: 0},
+      {bearing: 0, distance: 0, id: 0}],
```

Comment:
default distance needs to be set to -1

Reply:
I mean we can do that but has having it be 0 in the past been causing issues?

Reply:
@zwgold does state machine disregard 0 the same way it disregards -1?